config.example.json

```diff
@@ -15,7 +15,7 @@
 		"behindReverseProxy": false,
 		"blockRefreshInterval": "120ms",
 		"stateUpdateInterval": "3s",
-		"difficulty": 4000000000,
```

Comment:
Oh really? I thought JSON numbers were always decimal representations. TIL.

Still I don't understand this change. Now you've wrapped it as a JSON string instead of a number. Any reason? It seemed correct for this to be a number.